base/gfspvgmgr/sp_freeze_pool.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 const (
-	DefaultFreezingPeriodForSP = 5 * time.Minute
+	DefaultFreezingPeriodForSP = 30 * time.Minute
```

Comment:
what if too many SPs are in the frozen pool, and thus no enough SPs are available? Shall we unfreeze them in such case?

Reply:
How about checking the available SP onchain, and release if the diff is less than X.